ros-tutorials/lesson-1-basics.md

```diff
@@ -1,23 +1,35 @@
 # Lesson 1 - Basics
 
 ## Summary
-This lesson will introduce you into the very basics of ROS. By the end, you will know how to start ROS and run the turtle simulation. Before starting, you should take a few minutes and just read/watch videos about ROS in general, so you have a general understanding on what it is used for.
+This lesson will introduce you into the very basics of ROS. By the end, you will know how to start ROS and run the turtle simulation. Before starting, you should take a few minutes and just read/watch videos about ROS in general, just so that you have a general understanding of its uses.
```

Comment:
nit: you know how much I love **reading** those videos.

Reply:
Lol I was annoyed about that too but left it to keep it shorter. But since it bothered you, too, I added two words to make it make sense.